lenskit-core/src/main/java/org/lenskit/data/ratings/Rating.java

```diff
@@ -129,7 +153,9 @@ public long getTimestamp() {
      * returns null.
      *  
      * @return {code true} if there is a rating (the preference is non-null).
+     * @deprecated Unrates have gone away.
      */
+    @Deprecated
```

Comment:
How long are we keeping unrate code? In my mind we cut them out wholesale for LK3.0